newsroom/auth/forms.py

```diff
@@ -41,3 +41,10 @@ class LoginForm(FlaskForm):
 
 class TokenForm(FlaskForm):
     email = StringField('Email', validators=[DataRequired(), Length(1, 64), Email()])
+
+
+class ResetPasswordForm(FlaskForm):
+    old_password = PasswordField('Old password', validators=[])
```

Comment:
imo there should be no old password in reset process, that's the reason why you do it usually

Reply:
I see. I guess I was using ResetPassword form for both resetting (by a token) and for changing the password from user's My Details pages. Maybe it would be better if I separate them.

Reply:
Merging this for now but I'll revisit this area later on again.